README.md

```diff
@@ -13,7 +13,7 @@ You can find the [latest documentation](https://aggregation-caesar.zooniverse.or
 ---
```

Comment:
Do you have a list of what you needed to install?

Reply:
Sorry for the slow reply -- it was an issue with GTK3 library on Linux (which is apparently not a default installation). This would fix it on Ubuntu:
`sudo apt-get install build-essential libgtk-3-dev`